lib/collection/response.js

```diff
@@ -96,12 +96,265 @@ var util = require('../util'),
      */
     HTTP_X_X = 'HTTP/X.X ',
 
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    utf8 = 'utf-8',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    iso88591 = 'iso-8859-1',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    doubleQuotes = '"',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    $1 = '$1',
```

Comment:
Avoid $ special character start of variable name unless that serves a purpose. This could be `TOKEN_$1`